pcweb/components/sidebar/sidebar.py

```diff
@@ -243,11 +271,11 @@ def sidebar_category(name, icon, color, index):
         rx.button(
             rx.icon(
                 tag=icon,
-                color=rx.color(color, 1),
+                color="#fff",
```

Comment:
Can just be "white" or the latest gray color

Reply:
This needs to be white as it stays white in both dark and light mode